quipucords/api/migrations/0002_source_exclude_hosts.py

```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
why are we deleting this?

Reply:
Yeah, let's put the entire file back.

Reply:
It has to do with the migrations, I think it was accidentally added earlier.

Reply:
I think this was added by accident.

Reply:
This was deleted again

Reply:
Additionally, there will need to be a migration 3 for the use-paramiko.